corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
@esoergel, is this function ever called anywhere other than by `get_app`? If it is then @dannyroberts reasoning may be correct. If not, then I think @dannyroberts reasoning, while possibly correct, is not considering the entire context (see @esoergel's explanation: "`min_version` is only ever set if the app_id passed in corresponds to a released app").